library/templates/dob_now_applications.yml

```diff
@@ -27,5 +25,7 @@ dataset:
   info:
     description: |
       DOB NOW job applications received from DOB via FTP
+      the python handles the encoding of the source file which is not standard utf-8.
+      Also the versioning of this file is based on the date received from DOB
```

Comment:
Do we have an explanation for the dob now process in the wiki or written down somewhere? We receive this data via email from richard correct @AmandaDoyle?

Reply:
the most recent commit added more comments @mbh329 let me know if it should resolve this. Thanks!